### ISSUE TITLE
Add Cisco 1000v support

### ISSUE THREAD

**Issue by Thor192** [kentik/snmp-profiles#15]
Could we get support for Cisco 1000v routers?

SYSOID  .1.3.6.1.4.1.9.1.1537

Thanks!

**Comment by thezackm** [comment]
thanks @Thor192 - can you please upload a **SANITIZED** SNMP walk output so we can get this built for you? (It is important to take care and remove any sensitive information please)

An example walk can be seen here; simply replace `$COMMUNITY_STRING` and `$IP_ADDRESS` with your necessary values.
```
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS . >> cisco_csr_1000v.out
```

**Comment by Thor192** [comment]
Thank you, here you go.

[SNMPWALK1000v.txt](https://github.com/kentik/snmp-profiles/files/7140124/SNMPWALK1000v.txt)

**Comment by Thor192** [comment]
Dump. Is that file what is needed?

**Comment by thezackm** [comment]
@Thor192 - looks good thanks; I'll work on processing this and ping here if there are questions

**Comment by thezackm** [comment]
@Thor192 - can you please try SNMP walks against the following OIDs and copy the results from `STDOUT` back here?

```
# Processor CPU (control plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.2.1.58.0

# Platform CPU (data plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.5.7

# Processor Memory (control plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.48.1.1.1.5.1
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.48.1.1.1.6.1

# Platform Memory (data plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.12.7
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.13.7
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.27.7
```

**Comment by Thor192** [comment]
# Processor CPU (control plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.2.1.58.0
.1.3.6.1.4.1.9.2.1.58.0 = INTEGER: 1

# Platform CPU (data plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.8
.1.3.6.1.4.1.9.9.109.1.1.1.1.8.7 = Gauge32: 25

snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.5
.1.3.6.1.4.1.9.9.109.1.1.1.1.5.7 = Gauge32: 25

# Processor Memory (control plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.48.1.1.1.5
.1.3.6.1.4.1.9.9.48.1.1.1.5.1 = Gauge32: 254522656
.1.3.6.1.4.1.9.9.48.1.1.1.5.11 = Gauge32: 88

snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.48.1.1.1.6
.1.3.6.1.4.1.9.9.48.1.1.1.6.1 = Gauge32: 1866977612
.1.3.6.1.4.1.9.9.48.1.1.1.6.11 = Gauge32: 102316

# Platform Memory (data plane)
snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.12
.1.3.6.1.4.1.9.9.109.1.1.1.1.12.7 = Gauge32: 2488092

snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.13
.1.3.6.1.4.1.9.9.109.1.1.1.1.13.7 = Gauge32: 1490312

snmpwalk -v 2c -On -c $COMMUNITY_STRING $IP_ADDRESS .1.3.6.1.4.1.9.9.109.1.1.1.1.27
.1.3.6.1.4.1.9.9.109.1.1.1.1.27.7 = Gauge32: 3854984

Thanks!

**Comment by thezackm** [comment]
resolved with PR #17 

note I have a pending PR to be added into [newrelic/entity-definitions](https://github.com/newrelic/entity-definitions) that will create entities in New Relic. But right now you should be able to run a discovery against your CSR 1000v and return the device and then ship the data to New Relic. Confirm with the following: 

```
FROM Metric SELECT uniques(metricName) WHERE instrumentation.name = 'cisco-csr' AND instrumentation.provider = 'kentik'
```

**Comment by thezackm** [comment]
added newrelic/entity-definitions#315 to close this one out
